site/helm-integration.md

```diff
@@ -49,7 +49,10 @@ Flux-Helm Integration implementation consists of two parts:
 
  - name
  - namespace
- - chartGitPath ... path (from repo root) to a Chart subdirectory
+ - chartGitPath ... path (from repo root) to a Chart subdirectory or url to
```

Comment:
If supplying a URL to a git repo elsewhere works, it's a coincidence!

Reply:
Wow! Lighting fast feedback.  Are you saying this "feature" was not by design, it's just an accident/coincidence?

We indent to leverage this feature so we can have each service (which is each in a separate repo) provide their own helm chart in their respective repo so the charts do not have to be in the flux repo.

Should we leverage this feature or not?

Reply:
Yes, it's not intended to work that way. I imagine it works because the path ends up looking like an absolute URL, and Helm downloads it .. or something along those lines.

I would not rely on it. For one thing, it'll likely download everything from the git repo _a lot_.

Reply:
So are you saying currently we are stuck with having the all the different charts physically in the same flux repo?

This is really annoying.  We already have a few microservice that have their helm chart in their own respective repo.

Any other possible work-around to use charts from a remote git repo?

Reply:
In fact, no, pointing chartGitPath to a remote git url do not work.  Sorry, I did not test properly.

ts=2018-09-21T19:27:09.692439044Z caller=release.go:175 component=release error="Chart release failed: mongodb-database: &os.PathError{Op:\"stat\", Path:\"/tmp/flux-working999529096/charts/https:/github.com/tlvu/flux-helm-test/tree/master/charts/mongodb\", Err:0x2}"